doc/devops/docker-images.md

```diff
@@ -0,0 +1,55 @@
+# Docker images generated with CI
+
+Table of Contents
+=================
+
+   * [Dockerhub](#dockerhub)
+      * [Dockerhub vs Docker Cloud](#dockercloud)
+      * [When and what images are built](#images)
+      * [Dockerhub setup](#dockerhub-setup)
+         * [Creating/Configuring a new Automated Image](#new-image)
+         * [Collaborators &amp; teams](#collaborators-teams)
+
+[Dockerhub](https://hub.docker.com/) is the default Docker image repository managed and mantained by Docker Inc. In addition to the repository features, it also offers the possibility of automate the building of the Docker images using Github or Bitbucket triggers.
```

Comment:
consistent naming: it should be _Docker Hub_ and _Docker Cloud_ everywhere

Reply:
Agree